spec/services/page_cloner_spec.rb

```diff
@@ -31,6 +31,15 @@
     expect(cloned_page.campaign).to eq(campaign)
   end
 
+  it 'duplicates content' do
+    expect(cloned_page.content).to eq page.content
```

Comment:
I'd be more confident with this spec if it were testing against a real value: `expect(cloned_page.content).to eq("some content")` - the spec becomes meaningless if `page.content` is empty or `nil`.